src/project_info.rs

```diff
@@ -46,6 +46,12 @@ impl ProjectInfo {
 			whitelist.iter().any(|user| user == login)
 		})
 	}
+
+	pub fn is_admin(&self, login: &str) -> bool {
```

Comment:
Are all owners, delegated reviewers, and whitelisted devs admins? I would think that only `owner`s are admins.

Reply:
They all are

Reply:
It wouldn't be much use as a category if it were only the owner

Reply:
It's just to save us checking all three in some places

Reply:
Well I guess I more mean, I wouldn't expect that reviewers or whitelisted devs to also be admins, seems like they would only need `write` access not `admin`.

Reply:
It doesn't necessarily correspond to `admin` permissions.  Maybe it should be renamed.

Reply:
also the `delegated_reviewer` is a stand-in for the owner when they're away, distinct from requested reviewers.  maybe that should be renamed as well to `delegated_owner`.

Reply:
Yes if it was named something that makes it clear it's not `admin` permissions from GitHub I would be fine with that.